code/script.js

```diff
@@ -36,12 +41,103 @@ const showMessage = (message, sender) => {
 // Starts here
 const greetUser = () => {
   // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot')
+  showMessage("Konnichiwa Globetrotter! I'm an AI Travel Bot, what is your name?", 'bot')
   // Just to check it out, change 'bot' to 'user' here 👆
 }
 
 // Set up your eventlisteners here
 
+
+// Listen for the form submission event
+nameForm.addEventListener('submit', (event) => {
+  event.preventDefault(); // Prevent the form from submitting
+
+  if(userName == null){ // Check if userName is not already set
+  userName = nameInput.value; // Set userName from the input field
+  showMessage(userName, 'user'); // Display user name
+  nameInput.value = ''; // Clear the input field
+
+  // Display a welcome message from the bot after a short delay
+  setTimeout(() => {
+    const botMessage = `Hello ${userName}! Let's see the world!`;
+    showMessage(botMessage, 'bot');
+  }, 500);
+
+  // User options after a delay
+  setTimeout(() => {
+    const botMessage = `This is what I can do for you: 
+<p>1 - Book a flight</p>
+<p>2 - Book a hotel</p>
+<p>3 - Book a rental car</p>`;
+    showMessage(botMessage, 'bot');
+
+  // Instruct the user to select an option
+    setTimeout(() => {
+      const botMessage = `Please enter a number from 1 to 3 to select an option.`;
+      showMessage(botMessage, 'bot');
+    }, 2000);
+  }, 4000);
+} else{
+    const userChoice = nameInput.value; // Get user choice
+    showMessage(nameInput.value,'user') // Display user choice
+    nameInput.value = ''; // Clear the input field
+    let botReply = '';
+
+  // Bot reply based on user choice
+    switch (userChoice) {
+      case '1':
+        botReply = `Great! I have booked a one-way ticket for you to Samarkand, Uzbekistan. Total price €1439. The details will be sent via email. Have a nice trip! ✈️`;
+        break;
+      case '2':
+        botReply = `Good choice! A single room at City Inn Lodge in Botswana awaits you. I have booked 14 nights with breakfast included. Total price €2279. I wish you a pleasant stay! 🏨`;
+        break;
+      case '3':
+        botReply = `Perfect! A Ford Escape (or similar) is waiting for you in Cheyenne, ID. 9 days rental including free miles and insurance. Total price $1203. Get ready for some inspired driving! 🚗`;
```

Comment:
The price is in dollars here and in euros above maybe it could be good to have only one currency.